ui/diagnoseReportApp/translations/en.yaml

```diff
@@ -107,197 +107,197 @@ diagnosis:
-        etcd_wal_fsync: The time cost of etcd writing WAL into the persistent storage, the label is [instance]
-        pd_peer_round_trip: The latency of the network, the label is [To]
-        tikv_grpc_messge: The time cost of TiKV handle of gRPC message, the label is [type]
-        tikv_cop_request: The time cost of coprocessor handle read requests, the label is [req]
-        tikv_cop_handle: The time cost of handling coprocessor requests, the label is [req]
-        tikv_cop_wait: The time cost of coprocessor requests that wait for being handled, the label is [req]
-        tikv_scheduler_command: The time cost of executing commit command, the label is [type]
-        tikv_scheduler_latch_wait: The time cost of TiKV latch wait in commit command, the label is [type]
-        tikv_handle_snapshot: The time cost of handling snapshots, the label is [type]
-        tikv_send_snapshot: The time cost of sending snapshots, the label is [instance]
-        tikv_storage_async_request: The time cost of processing asynchronous snapshot requests, the label is [type]
-        tikv_raft_append_log: The time cost of Raft appends log, the label is [instance]
-        tikv_raft_apply_log: The time cost of Raft apply log, the label is [instance]
-        tikv_raft_apply_wait: The time cost of Raft apply wait, the label is [instance]
-        tikv_raft_process: The time cost of peer processes in Raft, the label is [instance]
-        tikv_raft_propose_wait: The wait time of each proposal, the label is [type]
-        tikv_raft_store_events: The time cost of raftstore events, the label is [type]
-        tikv_commit_log: The time cost of Raft commits log, the label is [instance]
-        tikv_check_split: The time cost of running split check, the label is [instance]
-        tikv_ingest_sst: The time cost of ingesting SST files, the label is [instance]
-        tikv_gc_tasks: The time cost of executing GC tasks, the label is [task]
-        tikv_pd_request: The time cost of TiKV sends requests to PD, the label is [type]
+        tidb_query: The time cost of SQL queries. The label is [sql_type].
+        tidb_get_token(us): The time cost of session getting token to execute the SQL query. The label is [instance].
+        tidb_parse: The time cost of parsing SQL queries. The label is [sql_type].
+        tidb_compile: The time cost of building the query plan. The label is [sql_type].
+        tidb_execute: The time cost of executing the SQL query, which does not include the time to get the results of the query. The label is [sql_type].
+        tidb_distsql_execution: The time cost of distsql execution. The label is [type].
+        tidb_cop: The processing time of KV storage Coprocessor. The label is [instance].
+        tidb_transaction: The time cost of durations when executing a transaction, including retry. The label is [sql_type].
+        tidb_transaction_local_latch_wait: The time cost of. The label is [instance].
```

Comment:
The time cost of what? The sentence is not finished.

Reply:
The time cost of local latch wait

Reply:
Updated in https://github.com/pingcap-incubator/tidb-dashboard/pull/461/commits/c6d827eabb77e609ea4a83052aaf091c606a00c7